README.md

```diff
@@ -146,6 +146,16 @@ RSpec::OpenAPI.response_headers = %w[X-Cursor]
 # Set `servers` - generate servers of a schema file
 RSpec::OpenAPI.servers = [{ url: 'http://localhost:3000' }]
 
+# Set `security_schemes` - generate security schemes
+RSpec::OpenAPI.security_schemes = {
```

Comment:
Better to add test.

Please set `security_schemas` in https://github.com/k0kubun/rspec-openapi/blob/master/spec/requests/rails_smart_merge_spec.rb and update the corresponding openapi files accordingly.